cmd/cvp-trigger/main.go

```diff
@@ -105,6 +107,7 @@ func (o *options) gatherOptions() {
 	fs.StringVar(&o.operatorPackageName, operatorPackageNameOptions, "", "Operator package name to test")
 	fs.StringVar(&o.releaseImageRef, releaseImageRefOption, "", "Pull spec of a specific release payload image used for OCP deployment.")
 	fs.StringVar(&o.targetNamespaces, targetNamespacesOption, "", "A comma-separated list of namespaces the operator will target. If empty, all namespaces are targeted")
+	fs.StringVar(&o.pyxisUrl, PyxisUrl, "", "Represents cvp product package name for specific ISV")
```

Comment:
Can we have same description as in the readme? `URL that contains specific cvp product package name for specific ISV with unique pid`

Reply:
Sure thing, modified.